sound/soc/intel/boards/sof_ssp_amp.c

```diff
@@ -152,12 +152,13 @@ sof_card_dai_links_create(struct device *dev, enum sof_ssp_codec amp_type,
 
 	/* codec SSP */
 	if (amp_type != CODEC_NONE) {
-		links[id].name = devm_kasprintf(dev, GFP_KERNEL, "SSP%d-Codec", ssp_codec);
-		if (!links[id].name)
+		be_id = fixed_be ? SPK_BE_ID : id;
```

Comment:
this still has the dead code, fixed_be is always true here, no?

Reply:
no. fixed_be is always true when HDMI-In is created.